quinn-proto/src/connection/mod.rs

```diff
@@ -2788,7 +2788,7 @@ where
         // Generate keys for the key phase after the one we're switching to, store them in
         // `next_crypto`, make the contents of `next_crypto` current, and move the current keys into
         // `prev_crypto`.
-        let new = self.crypto.next_1rtt_keys();
+        let new = self.crypto.next_1rtt_keys().unwrap(); // safe: only called for `Data` packets
```

Comment:
I feel like `unwrap()` + explanatory comment should usually be an `expect` with the rationale as a string. Does that seem reasonable you?

Reply:
Yup!